demo-shell/resources/i18n/de.json

```diff
@@ -285,104 +28,12 @@
       "XXTRALARGE": "8.Größe: extrem groß"
     }
   },
-  "SOCIAL": {
-    "LIKE": "Like-Komponente",
-    "RATING": "Bewertungskomponente"
-  },
-  "TAG": {
-    "LIST": "Content Services-Tagliste",
-    "INSERT": "Node-ID einfügen",
-    "NODE_LIST": "Tagliste nach Node-ID"
-  },
-  "DEMO_PERMISSION": {
-    "INHERIT_PERMISSION_BUTTON": "Berechtigungen erben",
-    "INHERITED_PERMISSIONS_BUTTON": "Berechtigungen geerbt"
-  },
-  "TASK_LIST_DEMO": {
-    "ERROR_MESSAGE": {
-      "APP_ID_REQUIRED_ERROR": "App-ID einfügen",
-      "APP_ID_TYPE_ERROR": "App-ID muss eine Zahl sein",
-      "NUMBER_TYPE_ERROR": "Wert muss Zahl sein",
-      "NUMBER_GREATER_THAN": "Wert muss größer oder gleich {{ value }} sein"
-    },
-    "TOOLTIP_MESSAGE": {
-      "START_INPUT": "Startseite"
-    },
-    "TASK_CONTEXT_MENU": "Aufgabenlisten-Kontextmenü"
-  },
-  "PROCESS_LIST_DEMO": {
-    "ERROR_MESSAGE": {
-      "APP_ID_REQUIRED_ERROR": "App-ID einfügen",
-      "APP_ID_TYPE_ERROR": "App-ID muss eine Zahl sein",
-      "NUMBER_GREATER_THAN": "Wert muss größer oder gleich {{ value }} sein"
-    },
-    "PROCESS_CONTEXT_MENU": "Prozesslisten-Kontextmenü"
-  },
   "GROUP-TITLE1-TRANSLATION-KEY": "Übersetzung benutzerdefinierter Titel 1",
   "GROUP-TITLE2-TRANSLATION-KEY": "Übersetzung benutzerdefinierter Titel 2",
   "ERROR_CONTENT": {
```

Comment:
Missed during cleaning?

Reply:
there are some e2e that expect those keys there, so decided not to deal with fixing tests so far. we can do that in the future iterations